en/docs/GettingStarted/overview.md

```diff
@@ -6,26 +6,28 @@ In an age where organizations are becoming more and more "data-driven", exposing
 
 ## **What is the WSO2 API Manager** ?
 
+WSO2 API Manager is the only 100% open source product that addresses all aspects of API management including API creation, consumption, security, rate limitation, monetization and analytics. The WSO2 API Manager was named as **a leader in "The Forrester Wave™: API Management Solutions, Q4 2018."**
+
 ![](/assets/attachments/103327648/103346653.png)
-In brief, the WSO2 API Manager consists of 6 main components, the [API Publisher,](_Design_API_) the [API developer Portal,](https://docs.wso2.com/display/SHAN/API+Developer+Portal) [API Gateway](_API_Gateway_) (now comes as a [API Microgateway](_API_Microgateway_) as well ), [Key Manager,](_API_Security_) [Traffic Manager](_Rate_Limiting_) and the [API Analytics](_Analytics_) . For more information on these components see [Architecture](https://docs.wso2.com/display/SHAN/Architecture) .
+In brief, the WSO2 API Manager consists of 6 main components, the [API Publisher,](#api-publisher) the [API developer Portal,](#developer-portal) [API Gateway](#api-gateway) (now comes as a [API Microgateway](_API_Microgateway_) as well ), [Key Manager,](#key-manager) [Traffic Manager](#traffic-manager) and the [API Analytics](#analytics) . For more information on these components see [Basic Architecture and Key Components]() .
```

Comment:
The serial comma is missing before "and"

Reply:
Content review will be done at a later stage.